java/flight/flight-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightConnection.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.calcite.avatica.AvaticaConnection;
 import org.apache.calcite.avatica.AvaticaFactory;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
Is there a more appropriate package for this annotation?

Reply:
We solved by using an XML filter instead.